hipcat/cli.py

```diff
@@ -88,19 +93,26 @@ def main(room, message, quote, code):
                     formatter='/quote' if quote else '/code',
                     message=message
                 )
+
             response = requests.post(
                 url,
                 headers={
                     'Authorization': 'Bearer {}'.format(config.hipchat_token),
                     'Content-Type': 'application/json',
                 },
                 data=json.dumps({
-                    'message': message
+                    'room_id': room,
+                    'message': message,
+                    'notify': notify,
+                    'from': sender,
+                    'color': color or 'yellow'
                 }),
-            ).json()
-            if 'error' in response:
-                print(response['error']['message'], file=sys.stderr)
-                exit(1)
+            )
+            if not notification:
```

Comment:
Do notifications return errors? If so, it would be good to handle them, instead of just assuming everything went okay.